src/core/services/gesture/gesture.spec.js

```diff
@@ -688,4 +688,36 @@ describe('$mdGesture', function() {
 
   });
 
+  describe('contenteditable', function() {
+
+    // Click tests should only be enabled when `$$hijackClicks == true` (for mobile)
+
+    it('should not hijack click on contenteditable element', inject(function($document, $mdGesture) {
+      if ( $mdGesture.$$hijackClicks ) {
```

Comment:
Where does this property exist? Never saw this one in our gesture service.

Also the tests should enable hijacking by default.